### ISSUE TITLE
Unable to run introduction on teach.cs

### ISSUE THREAD

**Issue by jasonkclee** [alecjacobson/geometry-processing-introduction#70]
I cloned the repo on the teach.cs machine and tried to run it there.
I followed the build instructions which worked fine, but when running introduction, the main.cpp code runs up until the viewer tries to open. No window shows up, and this message appears in the terminal:

GLX: GLX version 1.3 is requiredintroduction: /h/u5/c7/05/leejas45/Desktop/csc419/geometry-processing-introduction/libigl/external/glfw/src/window.c:475: glfwWindowShouldClose: Assertion `window != NULL' failed.

**Comment by chanlenn** [comment]
I don't think you should be able to run this remotely, since you only have access to the terminal.

**Comment by jasonkclee** [comment]
I am using the NX client (you'd have access to the full GUI and OS, and opening apps like FIrefox; you're just accessing it remotely). So I should still be able to run the application/expect a window to open there.

**Comment by alecjacobson** [comment]
I don't believe NX will work (for obscure reasons), however, virtualgl
should work and is supported by teach.cs here's their info page
https://www.teach.cs.toronto.edu/using_cdf/vgl.html

(I'd love to know either way if this works or not)

>
